package.json

```diff
@@ -64,6 +64,7 @@
     "@types/jest": "^26.0.19",
     "eslint": "^7.20.0",
     "husky": "4.3.5",
+    "@types/react-redux": "^7.1.16",
```

Comment:
Couldn't figure out why you auto importing `useDispatch` wasn't working - this was the reason.